pkg/kv/kvserver/BUILD.bazel

```diff
@@ -319,6 +319,7 @@ go_test(
         "txn_recovery_integration_test.go",
         "txn_wait_queue_test.go",
     ],
+    args = ["-test.timeout=3595s"],
```

Comment:
@healthy-pod I am curious how these timeout values were derived?

Reply:
[Bazel sets the test timeout based on the test size](https://bazel.build/reference/test-encyclopedia#role-test-runner). If the test size is not explicitly set then it will be `medium`. The default mappings are:
```
small: 60s timeout
medium: 300s timeout
large: 900s timeout
enormous: 3600s timeout
```

We let the `go test` process timeout 5 seconds before bazel kills it and that's how the quoted `3595s` (enormous) was derived.